scripts/spelling.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
```

Comment:
As sort of mentioned before, I'm beginning to wonder more if this should all become ci.sh or part of the Makefile or something, I'm not really sure. Probably makes sense to leave it until everything is migrated so we can easily re-assess what it all looks like compared to .travis-ci.sh .

Reply:
Yeah that's cool, lets leave this for the future then.